README.md

```diff
@@ -18,6 +18,21 @@ While these are often well suited for in-system communication, they often cause
 Those connections may also have additional requirements, like low bandwidth, high latency, etc., or custom communication hardware, which does not have an ethernet stack.
 This library defines an interface to those robots and supports externally programmed Transports that can handle the requirements above.
 
+## Citing
+
+An evaluation of library has beed published at the i-SAIRAS conference in 2020, you can find the Paper [here](https://www.dfki.de/web/forschung/publikationen/renameFileForDownload?filename=I-SAIRAS_2020-RRC_library-DFKI.pdf&file_id=uploads_4831).
```

Comment:
Minor correction: An evaluation of **the** library

Secondly. The current link is a direct download link. I think it is preferable to view the paper first. Therefore I'd choose the original i-Sairas link:

https://www.hou.usra.edu/meetings/isairas2020fullpapers/pdf/5063.pdf